setup.cfg

```
@@ -74,7 +74,7 @@ testpaths = "jdaviz" "docs"
 astropy_header = true
 doctest_plus = enabled
 text_file_format = rst
-addopts = --doctest-rst -Wignore
+adopts = --doctest-rst -Wignore
```

Why change this? https://docs.pytest.org/en/6.2.x/customize.html

```suggestion
addopts = --doctest-rst -Wignore
```

---

`pytest .` does not work when `addopts` is used. Not sure why that is happening but it stopped when changed to `adopts`, so I thought it might be a typo. I can change it back but am I the only one with `pytest` issues?

---

Try change it back and see if the CI crashes. The traceback would provide clues.